content/blog/2016/2016-08-16-jenkins-world-speaker-blog-aquient.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Jenkins World Speaker Highlight: Continuously Delivering Continuous Delivery Pipelines"
+tags:
+- event
+- JenkinsWorld
+author: hinman55
+---
+
+NOTE: This is a guest post by Jenkins World speaker Neil Hunt, Senior DevOps Architect at link:http://www.aquilent.com/[Aquilent].
+
+In smaller companies with a handful of apps and fewer silos, implementing CD pipelines to support these apps is fairly straightforward using one of the many delivery orchestration tools available today. There is likely a constrained tool set to support - not an abundance of flavors of applications and security practices - and generally fewer cooks in the kitchen. But in a larger organization, I have found that in the past, there were seemingly endless unique requirements and mountains to climb to reach this level of automation on each new project.  
+
+Enter the Jenkins Pipeline plugin. My recently departed former company, a large financial services organization with a 600+ person IT organization and 150+ application portfolio, set out to implement continuous delivery enterprise-wide. After considering several pipeline orchestration tools, we determined the Pipeline plugin (at the time called Workflow) to be the superior solution for our company. Pipeline has continued Jenkins' legacy of presenting an extensible platform with just the right set of features to allow organizations to scale its capabilities as they see fit, and do so rapidly. As early adopters of Pipeline with a protracted set of requirements, we used it both to accelerate the pace of onboarding new projects and to reduce the ongoing feature delivery time of our applications.  
+
+In my presentation at Jenkins World, I will demonstrate the methods we used to enable this. A few examples:  
+
+* We leveraged the Pipeline Remote File Loader plugin to write shared common code and sought and received community enhancements to these functions.  
+
+image:/images/post-images/jw-speaker-blog-aquient/jw-speaker-blog-aquilent-1.png[role=left]
+
+_Jenkinsfile, loading a shared AWS utilities function library_
+
+image:/images/post-images/jw-speaker-blog-aquient/jw-speaker-blog-aquilent-2.png[role=left]
```

Comment:
These images should be "role=center" otherwise they will cause text to wrap around the right side of them like so:

![text-wrap](https://cloud.githubusercontent.com/assets/26594/17741380/cdae0e42-6450-11e6-9d5a-82a8969d6c9a.png)